### DISCUSSION TITLE
Add support for scroll area

### DISCUSSION THREAD

**Discussion by crazyair** [exceljs/exceljs#1721] [Ideas & Proposals]
## 💬 Questions and Help

Please do not use issues for making a questions, for this purposes much better is GitHub Discussions feature:
https://github.com/exceljs/exceljs/discussions/categories/q-a

### How to make a question?

![image](https://user-images.githubusercontent.com/7971419/119211283-ccb9c000-bae3-11eb-8d72-c659a7916912.png)

**Comment by Siemienik** [comment]
AFAIK, no. That why I moved it into `Ideas & Proposals`. 

If this feature is important for your project, it is possible to make an order on consulting@siemienik.pl .

Kind regards
